.github/workflows/ci.yaml

```diff
@@ -42,10 +43,20 @@ jobs:
           libboost-dev libboost-chrono-dev libboost-random-dev libboost-system-dev \
           libqt5svg5-dev qtbase5-dev qttools5-dev
 
-    - name: install libtorrent from source
+    - name: get libtorrent source code
       run: |
         git clone https://github.com/arvidn/libtorrent && cd libtorrent
-        git checkout ${{ env.LIBTORRENT_VERSION_TAG }}
```

Comment:
@Chocobo1 

I was expecting https://github.com/qbittorrent/qBittorrent/pull/15093/files to be merged first before rebasing to make that change, since it would be easier.

Could you please provide feedback there? Thanks.

Reply:
@Chocobo1 rebased, fixed.